interface/requests/response.go

```diff
@@ -6,172 +6,187 @@ package requests
-	RespKeyDoesNotExist              uint16 = 15
-	RespKeyAlreadyExists             uint16 = 16
-	RespPsaErrorGenericError         uint16 = 1132
-	RespPsaErrorNotPermitted         uint16 = 1133
-	RespPsaErrorNotSupported         uint16 = 1134
-	RespPsaErrorInvalidArgument      uint16 = 1135
-	RespPsaErrorInvalidHandle        uint16 = 1136
-	RespPsaErrorBadState             uint16 = 1137
-	RespPsaErrorBufferTooSmall       uint16 = 1138
-	RespPsaErrorAlreadyExists        uint16 = 1139
-	RespPsaErrorDoesNotExist         uint16 = 1140
-	RespPsaErrorInsufficientMemory   uint16 = 1141
-	RespPsaErrorInsufficientStorage  uint16 = 1142
-	RespPsaErrorInssuficientData     uint16 = 1143
-	RespPsaErrorCommunicationFailure uint16 = 1145
-	RespPsaErrorStorageFailure       uint16 = 1146
-	RespPsaErrorHardwareFailure      uint16 = 1147
-	RespPsaErrorInsufficientEntropy  uint16 = 1148
-	RespPsaErrorInvalidSignature     uint16 = 1149
-	RespPsaErrorInvalidPadding       uint16 = 1150
-	RespPsaErrorTamperingDetected    uint16 = 1151
+	StatusSuccess                      StatusCode = 0
+	StatusWrongProviderID              StatusCode = 1
+	StatusContentTypeNotSupported      StatusCode = 2
+	StatusAcceptTypeNotSupported       StatusCode = 3
+	StatusVersionTooBig                StatusCode = 4
+	StatusProviderNotRegistered        StatusCode = 5
+	StatusProviderDoesNotExist         StatusCode = 6
+	StatusDeserializingBodyFailed      StatusCode = 7
+	StatusSerializingBodyFailed        StatusCode = 8
+	StatusOpcodeDoesNotExist           StatusCode = 9
+	StatusStatusonseTooLarge           StatusCode = 10
+	StatusUnsupportedOperation         StatusCode = 11
+	StatusAuthenticationError          StatusCode = 12
+	StatusAuthenticatorDoesNotExist    StatusCode = 13
+	StatusAuthenticatorNotRegistered   StatusCode = 14
+	StatusKeyDoesNotExist              StatusCode = 15
+	StatusKeyAlreadyExists             StatusCode = 16
+	StatusPsaErrorGenericError         StatusCode = 1132
+	StatusPsaErrorNotPermitted         StatusCode = 1133
+	StatusPsaErrorNotSupported         StatusCode = 1134
+	StatusPsaErrorInvalidArgument      StatusCode = 1135
+	StatusPsaErrorInvalidHandle        StatusCode = 1136
+	StatusPsaErrorBadState             StatusCode = 1137
+	StatusPsaErrorBufferTooSmall       StatusCode = 1138
+	StatusPsaErrorAlreadyExists        StatusCode = 1139
+	StatusPsaErrorDoesNotExist         StatusCode = 1140
+	StatusPsaErrorInsufficientMemory   StatusCode = 1141
+	StatusPsaErrorInsufficientStorage  StatusCode = 1142
+	StatusPsaErrorInssuficientData     StatusCode = 1143
+	StatusPsaErrorCommunicationFailure StatusCode = 1145
+	StatusPsaErrorStorageFailure       StatusCode = 1146
+	StatusPsaErrorHardwareFailure      StatusCode = 1147
+	StatusPsaErrorInsufficientEntropy  StatusCode = 1148
+	StatusPsaErrorInvalidSignature     StatusCode = 1149
+	StatusPsaErrorInvalidPadding       StatusCode = 1150
+	StatusPsaErrorTamperingDetected    StatusCode = 1151
```

Comment:
You might want to double check the PSA ones as well, we changed them from what they were at the beginning!

Reply:
re-created whole lot from docs